server/pulp/server/webservices/views/consumer_groups.py

```diff
@@ -382,13 +383,12 @@ def verify_group_resources(group_id, repo_id, distributor_id):
     """
     missing_resources = {}
     group_manager = factory.consumer_group_query_manager()
-    repo_manager = factory.repo_query_manager()
     distributor_manager = factory.repo_distributor_manager()
     try:
         group_manager.get_group(group_id)
     except pulp_exceptions.MissingResource:
         missing_resources['group_id'] = group_id
-    repo = repo_manager.find_by_id(repo_id)
+    repo = model.Repository.objects(repo_id).first()
```

Comment:
This is one of a few places I have seen recently that use this pattern; rather than raising the first exception, it continues on so that a list of all invalid params can be raised together. I know this isn't your favorite pattern, but I don't think it would be appropriate to change it as a part of this story.

In order to preserve this, I thought it would be easier to read if I used `.first()` to allow `repo` to be `None` rather than using a try-except block.